tree/ntuple/v7/inc/ROOT/RFieldToken.hxx

```diff
@@ -36,8 +42,9 @@ It can be used for fast indexing in REntry's methods, e.g. REntry::BindValue().
 class RFieldToken {
    friend class REntry;
    friend class RNTupleModel;
+   friend class Experimental::Detail::RRawPtrWriteEntry;
 
-   std::size_t fIndex = 0;                      ///< The index in `fValues` that belongs to the field
+   std::size_t fIndex = 0;                      ///< The index of the top-level field
```

Comment:
With the addition of `REntry::RegisterSubfield`, this doesn't necessarily only have to be a top-level field.